WORKSPACE

```diff
@@ -0,0 +1 @@
+workspace(name = "com_googlesource_code_gflags")
```

Comment:
I don't think a library such as gflags which may rather be a external dependency of another project, i.e., being one "package" within the workspace of a Bazel build should contain such `WORKSPACE` file. A Bazel "package" only requires a `BUILD` file.

If the gflags library should be built and installed as separate package, i.e., not as part of a Bazel build of an application that depends on this library, you should use CMake instead.